tests/test_commands.py

```diff
@@ -330,6 +331,22 @@ def test_invalid_dataset_error_cases(data_model, dataset, exception_message):
     )
 
 
+def test_validate_no_db():
```

Comment:
`test_validate_no_db` or test no csv folder?

Reply:
This is to test mostly that the Validate folder without database properly raises exception.